pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -129,29 +125,6 @@
 
   private CommandLine commandLine;
 
-  public static class RpcApisConverter implements ITypeConverter<RpcApi> {
```

Comment:
Moving these classes out is orthagonal to this PR. Please do this in another PR.

Reply:
Back to original place